docs/sandbox/VehicleRentalServiceDirectory.md

```diff
@@ -1,24 +1,95 @@
+| language                                            |     `string`    | Language code.                                                             | *Optional* |               |  2.1  |
+| sourcesName                                         |     `string`    | Json tag name for updater sources.                                         | *Optional* | `"systems"`   |  2.1  |
+| updaterNetworkName                                  |     `string`    | Json tag name for the network name for each source.                        | *Optional* | `"id"`        |  2.1  |
+| updaterUrlName                                      |     `string`    | Json tag name for endpoint urls for each source.                           | *Optional* | `"url"`       |  2.1  |
+| url                                                 |      `uri`      | Endpoint for the VehicleRentalServiceDirectory                             | *Required* |               |  2.1  |
+| [headers](#vehicleRentalServiceDirectory_headers)   | `map of string` | HTTP headers to add to the request. Any header key, value can be inserted. | *Optional* |               |  2.1  |
+| [networks](#vehicleRentalServiceDirectory_networks) |    `object[]`   | List all networks to include.                                              | *Optional* |               |  2.4  |
+|       geofencingZones                               |    `boolean`    | Enables geofencingZones for the given network                              | *Optional* | `false`       |  2.4  |
+|       network                                       |     `string`    | The network name                                                           | *Required* |               |  2.4  |
+
+<!-- PARAMETERS-TABLE END -->
+
+
+### Parameter Details
+
+<!-- PARAMETERS-DETAILS BEGIN -->
+<!-- NOTE! This section is auto-generated. Do not change, change doc in code instead. -->
+
+<h4 id="vehicleRentalServiceDirectory_headers">headers</h4>
+
+**Since version:** `2.1` ∙ **Type:** `map of string` ∙ **Cardinality:** `Optional`   
+**Path:** /vehicleRentalServiceDirectory 
+
+HTTP headers to add to the request. Any header key, value can be inserted.
+
+<h4 id="vehicleRentalServiceDirectory_networks">networks</h4>
+
+**Since version:** `2.4` ∙ **Type:** `object[]` ∙ **Cardinality:** `Optional`   
+**Path:** /vehicleRentalServiceDirectory 
+
+List all networks to include.
+
+
+<!-- PARAMETERS-DETAILS END -->
+
+
+### Example
+
+<!-- JSON-EXAMPLE BEGIN -->
+<!-- NOTE! This section is auto-generated. Do not change, change doc in code instead. -->
+
+```JSON
+// router-config.json
+{
+  "vehicleRentalServiceDirectory" : {
+    "url" : "https://example.com",
+    "sourcesName" : "systems",
+    "updaterUrlName" : "url",
+    "updaterNetworkName" : "id",
+    "headers" : {
+      "ET-Client-Name" : "otp"
+    },
+    "networks" : [
```

Comment:
All networks in the directory are added, with the default of `geofencingZones: false`, and only those networks where you want to override this setting must be listed in `networks`. Am I reading this correctly?

Reply:
I have added a default network to set the default values:
```
"networks": [
      { "network": "default-network", "geofencingZones" : false },
      { "network": "boltdrammen", "geofencingZones" : true }
    ]
```